newdle/client/src/components/Home.js

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
-import {useSelector} from 'react-redux';
+import {useDispatch, useSelector} from 'react-redux';
 import {Link} from 'react-router-dom';
 import {isLoggedIn} from '../selectors';
+import {setStep} from '../actions';
 
 export default function Home() {
+  const dispatch = useDispatch();
   const isUserLoggedIn = useSelector(isLoggedIn);
 
   return (
     <>
       <h2>Home</h2>
-      {isUserLoggedIn && <Link to="/new">Create newdle</Link>}
+      {isUserLoggedIn && (
+        <Link to="/new" onClick={() => dispatch(setStep(1))}>
```

Comment:
Not sure this belongs here. What about dispatching an action that clears *all* creation-related data from the state instead (I think right now the participant list etc. isn't reset), and, since resetting the step would show the wrong page, redirect to something like `/created` after creation and show the success message etc. there? (in that case it now does make sense to put some data about to the newly created newdle in the redux store)

Reply:
If I set the step after successful creation wouldn't the app transition from step 3 (where are are looking at the success page with the event url) to step 1?

Reply:
> since resetting the step would show the wrong page, redirect to something like /created after creation and show the success message etc. there

Reply:
I see you already replied in that message